control/control.openSUSE.xml

```diff
@@ -1086,10 +1086,6 @@ TODO: prepare disk here when it does not break space calculation (and remove bel
                     </arguments>
                     <retranslate config:type="boolean">true</retranslate>
                 </module>
-                <module>
-                    <label>Network Activation</label>
```

Comment:
@lkocman @jreidinger looks like you removed the client by error or was it removed intentionally?

We are adding it back for Leap15.4 (#242 )